### PR TITLE
Automation for CEPH-83594298

### PR DESCRIPTION
**Pull request by aarsharm**
# Description

Pass logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-Y5F2XO

### CODE REVIEW

File: cli/rbd/group.py [hunk 1]
```diff
@@ -49,6 +49,23 @@ def list(self, **kw):
 
         return self.execute_as_sudo(cmd=cmd)
 
+    def info(self, **kw):
```
Comment (sunilangadi2) [inline cli/rbd/group.py:52]: line 233 also with `info` method?

File: tests/rbd/test_rbd_groups_image_clone.py [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+    try:
+        if kw.get("client_node"):
+            client = get_node_by_id(kw.get("ceph_cluster"), kw.get("client_node"))
+        else:
+            client = kw.get("ceph_cluster").get_nodes(role="client")[0]
+        kw["do_not_create_image"] = True
+        rbd_obj = initial_rbd_config(**kw)
+        pool_types = rbd_obj.get("pool_types")
+        ret_val = test_rbd_groups_image_clone(rbd_obj=rbd_obj, client=client, **kw)
```
Comment (sunilangadi2) [inline tests/rbd/test_rbd_groups_image_clone.py:231]: please add `log.info` for the test case pass

File: cli/rbd/group.py (hunk 1 above)
Comment (aarsharm) [reply cli/rbd/group.py:52]: Line 233 info is for Snap class.
The info function i added is for Group Class

**Review by sunilangadi2 — DISMISSED** [review]
changes LGTM.

File: tests/rbd/test_rbd_groups_image_clone.py [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+        log.error(
+            f"Testing RBD cloning image from group snapshot failed with Error: {str(e)}"
+        )
+        ret_val = 1
+    finally:
+        cluster_name = kw.get("ceph_cluster", {}).name
+        if "rbd_obj" not in locals():
+            rbd_obj = Rbd(client)
+        obj = {cluster_name: rbd_obj}
+        cleanup(pool_types=pool_types, multi_cluster_obj=obj, **kw)
```
Comment (HaruChebrolu) [inline tests/rbd/test_rbd_groups_image_clone.py:244]: Here if pool_types is not defined (e.g., due to a failure in initial_rbd_config()), the cleanup call will throw an UnboundLocalError.
Suggestion: Initialize pool_types to an empty list or include a check before cleanup:

Comment (aarsharm) [reply tests/rbd/test_rbd_groups_image_clone.py]: Done

**Review by HaruChebrolu — APPROVED** [review]
LGTM

**Pull request merged**